### PR TITLE
Fixed a few typos

### PR DESCRIPTION
**Pull request by DominikTo**

### CODE REVIEW

File: lib/Component/VCard.php [hunk 1]
```diff
@@ -357,7 +357,7 @@ function getValidationRules() {
 
             // FN is commented out, because it's already handled by the
             // validate function, which may also try to repair it.
-            // 'FN'           => '+',
+            // 'FN'           => '1',
```
Comment (evert) [inline lib/Component/VCard.php]: Are you sure about this?

Comment (evert) [inline lib/Component/VCard.php]: Nope it's wrong. http://tools.ietf.org/html/rfc6350#section-6.2.1 says:

    Cardinality:  1*

Which means 'at least one, but can be more.'

Comment (DominikTo) [inline lib/Component/VCard.php]: Haha, isn't it kinda late (or early)? 😴

The RFC [says](https://tools.ietf.org/html/rfc6350#section-6.2.1) *1 ("One **or more** instances per vCard MUST be present"), so according to the RFC "+" is correct, but it didn't make sense to me. Why would there ever be more than one FN?

Aren't we also enforcing just 1 FN here?
https://github.com/fruux/sabre-vobject/blob/master/lib/Component/VCard.php#L269

Happy to revert this one though =) Let me know.

Comment (evert) [inline lib/Component/VCard.php]: If I recall correctly, one use-case for multiple `FN` properties is different language variations. There's a `LANG` property as well for that purpose.

Comment (evert) [inline lib/Component/VCard.php]: And you are right that validator function is wrong.

Comment (DominikTo) [inline lib/Component/VCard.php]: Ah, that makes sense then. Rolling back this one.

**Pull request merged**